src/components/event/MapModal/Map-modal.js

```diff
@@ -61,44 +57,43 @@ class MapModal extends Component {
                         }
                         <div>
                             {
-                                this.props.values &&
-                                this.props.values.selectedPos != undefined &&
-                                this.props.values.selectedPos.latitude != undefined &&
-                                this.props.values.selectedPos.longitude != undefined &&
+                                // values &&
+                                // values.selectedPos != undefined &&
+                                values?.selectedPos?.latitude !== undefined &&
+                                values?.selectedPos?.longitude !== undefined &&
```

Comment:
Please verify if we can simplify these conditions to not explicitly check for `null` or `undefined` values